apps/vr-tests-react-components/src/stories/Spinner.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30592&commitId=5de861233b0874bae42b6b3e58d13243f6f3a64b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Spinner converged</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|384| Changed|
|Spinner converged.Inverted - High Contrast.chromium.png|150| Changed|
|Spinner converged.Primary - Dark Mode.chromium.png|59| Changed|
|Spinner converged.Primary - High Contrast.chromium.png|179| Changed|
|Spinner converged.Inverted.chromium.png|96| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|715| Changed|
|Spinner converged.Primary with Label.chromium.png|356| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|96| Changed|
|Spinner converged.Primary with Label - RTL.chromium.png|358| Changed|
|Spinner converged.Primary.chromium.png|89| Changed|
|Spinner converged.Inverted with Label.chromium.png|384| Changed|
|Spinner converged.Primary + size.chromium.png|620| Changed|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|235| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|600| Changed|
|Spinner converged.Animation.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>